tests/Cache/CacheManagerTest.php

```diff
@@ -319,9 +321,35 @@ public function testMakesRepositoryWithoutDispatcherWhenEventsDisabled()
         $this->assertNull($repoWithoutEvents->getEventDispatcher());
     }
 
+    public function testConcatSessionIdToPrefix()
+    {
+        $session = m::mock(Store::class);
+        $session->shouldReceive('getId')->andReturn('my_session_id');
+
+        $userConfig = [
+            'cache' => [
+                'stores' => [
+                    'session' => [
+                        'driver' => 'redis',
```

Comment:
Why would anyone ever do this?

Reply:
I don't know why anyone would do this, but I can remove it.

Reply:
yeah I would remove this and also then remove the applySessionPrefixIfNecessary thing that supports this.